include/nihstro/shader_bytecode.h

```diff
@@ -663,6 +663,13 @@ union SwizzlePattern {
         w = 3
     };
 
+    unsigned GetRawSelector(unsigned src) const {
```

Comment:
Can we have some documentation on this, since it's probably not too obvious that "src" is one-based, for instance?